packages/content/templates/nuxt-content.js

```diff
@@ -53,7 +59,7 @@ function propsToData (node, doc) {
       obj[attribute] = value
     }
     return data
-  }, { attrs: {} })
+  }, { class: null, style: null, attrs: {} })
```

Comment:
It is good to set initial value `null`, for Vue not to render "empty" css class (e.g. `<p class=""></p>`).

Reply:
This is strange, Vue should not create empty `class` attribute when there is no value for it.  
Do you mind to create a simple reproduction for this issue?

Reply:
@farnabaz 

Here is a [demo](https://codesandbox.io/s/vue-class-attr-rendering-3hdc4), and here is [the related part in the official doc](https://v3.vuejs.org/guide/migration/attribute-coercion.html#attribute-coercion-behavior).

Reply:
Thank you for the demo, but I mean a reproduction with nuxt-content.

I agree that having `class: ''` cause creating and empty attribute. 
Having null as default value for class and style is not necessary, and it should work without these defaults.

I suggest to remove these defaults and create another PR/issue with a reproduction sample that point the issue with nuxt-content

Reply:
@farnabaz 

I do appreciate if you take a look at my last comment and commit. 🙏 

> I pushed another commit to remove default values of class and style.